docs/source/mc_model.rst

```diff
@@ -3,71 +3,58 @@
 MC Model
 ========
 
-In this section you find reference documentation for the modules
-related to the MC Model.
-
-The simtools description of the MC Model has the :ref:`telescope_model`
+The gammasim-tools description of the MC Model has the `telescope_model`_
 module (and its main class TelescopeModel) as its central element.
 The TelescopeModel class provides a representation of a telescope in terms
-of the MC Model. A TelescopeModel is basically defined by its telescope name.
+of the MC Model. A TelescopeModel is defined by its telescope name.
 An interface with the :ref:`Model Parameters DB`
 is provided so the model parameters can be read and exported as configuration
 files.
 
-For the sake of modularity, lower level modules are defined to describe
-sub-elements of the TelescopeModel. At the moment, these modules are
-:ref:`mirrors` and :ref:`camera`.
-
-Array of telescopes is described by the Array Model module (not implemented yet),
-which contains a collection of TelescopeModel's and the array layout.
-
+Lower level modules are defined to describe sub-elements of the TelescopeModel.
+These modules are `mirrors`_ and `camera`_.
```

Comment:
but also array_model, telescope_model, and model utilities

Reply:
But these are not sub-elements of the TelescopeModel - we explicitly discuss those here. How can we make this clearer in this sentence, do you have a suggestion?

Reply:
Ah ok. So, I think the TelescopeModel is then not the central part of the MC Model as it is written in the text, but it is one of the central parts? Both the simtools/model repository and the list in mc_model.rst have 5 classes, so I don't see why we should only mention two in the text. Am I missing something?

Reply:
But wait, we have in this file:
- first paragraph: telescope model as central element
- second paragraph: mirrors and cameras as sub-elements of the telescope model
- third paragraph: array_model as collection of telescope models (I've fixed the missing link here)

What is not mentioned is the modul_utils module, but I am note sure if it is needed. The text in the paragraph has the intention to introduce the logic of the three bullet points above.

Reply:
Ok, I am convinced now, this is good. thanks for clarifying it

Reply:
OK, forget it, I misunderstood the meaning of module here.

Reply:
@VictorBarbosaMartins - I have complete rewritten and shortened that paragraph now to make it clearer. Could you have another look?